src/screens/ChannelListScreen/index.tsx

```diff
@@ -16,6 +16,7 @@ import useRootChannelListHook from '../../hooks/screen/useRootChannelListHook';
 import useSignedChannelListScreenHook from '../../hooks/screen/useSignedChannelListHook';
 import Search from './elements/Search';
 import useFollowUser from './hooks/useFollowUser';
+import {COLORS} from '../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `COLORS` has been added. Ensure that the `COLORS` object is properly exported from the `../../utils/theme` module and contains the color properties you intend to use in this component.
```diff
+ import {COLORS} from '../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->